### PR TITLE
WIP: docs: update readme

### PR DESCRIPTION
**Pull request by nazreen**

### CODE REVIEW

**Comment by nazreen** [comment]
@lilsweetcaligula this is a WIP. check your slack message.

**Comment by nazreen** [comment]
@lilsweetcaligula based on our most recent exchange, I have pushed a new commit

File: README.md [hunk 1]
```diff
@@ -1,18 +1,41 @@
...
-export GOOGLE_APPLICATION_CREDENTIALS=path/to/key.json
-
 ## Up and Running
 
+### Notes
+
+API calls require a token and email in the header
+(perhaps we should phase out needing the email in header as the decoded token might contain the email. TODO: check if decoded token contains email.)
+
+### Steps
```
Comment (lilsweetcaligula) [inline README.md]: Two more steps:

```
OBTAINING A SERVICE ACCOUNT KEY
- Request a senior Open Promises member to generate a service account key for you
- Place the key under `${OP_API_PROJECT_ROOT}/functions/secrets/google-key.json`, where `OP_API_PROJECT_ROOT` is your local path to the OP API repo.
```

```
INSTALL THE GOOGLE CLOUD SDK
- Go to https://cloud.google.com/sdk/install
- Follow the instructions to install the Google Cloud SDK
- Run the following command in your terminal: `$ gcloud auth application-default login`

If you ignore this step, the OP API will throw an error: `Error: Could not load the default credentials.` whenever you send a request to it.
```

File: README.md [hunk 2]
```diff
@@ -1,18 +1,41 @@
...
+
+RUN THE API SERVER
+
 run `cd functions/`
+run `npm i`
 run `npm run tsc:watch` and leave terminal open
-in a new terminal `npm run serve-h`
-perform http query`GET localhost:5000/promises`
+in a new terminal, run `npm run serve-h`
+perform http query`GET localhost:5000/promises` with these headers set:
```
Comment (lilsweetcaligula) [inline README.md]: I believe this step is a bit incorrect, the exact URL must be obtained from the terminal where the `npm run serve-h` command had been run prior.

**Comment by lilsweetcaligula** [comment]
Submitted a PR https://github.com/OpenPromisesProject/op-serverless-api/pull/148 for review based on this one.

File: README.md [hunk 3]
```diff
@@ -1,18 +1,41 @@
...
+
+API calls require a token and email in the header
+(perhaps we should phase out needing the email in header as the decoded token might contain the email. TODO: check if decoded token contains email.)
+
+### Steps
+
+GET YOUR FIREBASE TOKEN
+go to openpromises.com
+log out if already logged in
+sign in
```
[the remark this answers is not in the document]
Comment (nazreen) [reply README.md]: good spot. will amend.

File: README.md (hunk 1 above)
Comment (nazreen) [reply README.md]: thank you! I'll copy this and add them to the first part 'prerequisites'

File: README.md (hunk 2 above)
Comment (nazreen) [reply README.md]: you're right

**Comment by nazreen** [comment]
> Submitted a PR #148 for review based on this one.

oh man I only read this now. ok I'll merge that one instead. thank you

**Pull request closed without merging**